internal/health/health_topsuite_test.go

```diff
@@ -0,0 +1,58 @@
+package health
+
+import (
+	"context"
+	"testing"
+
+	"github.com/G-Research/unicorn-history-server/test/database"
+	"github.com/jackc/pgx/v5/pgxpool"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type HealthToptSuite struct {
```

Comment:
I don't understand this naming - specifically, the "Topt" part of "HealthToptSuite" - what does that signify?

Reply:
That's a good catch, Wanted to write `HealthTopSuite` but misspelled! Still not satisfied with the name though. Let me know if you can think of a better name 🤞

Reply:
How about just `HealthSuite`?  Its location in the hierarchy hints that it will be running sub-suites, etc.

Reply:
Yeah make sense, updated